libkrun/test/test_libkruntime.py

```diff
@@ -83,6 +83,21 @@ def test_cycles_u64_double_ratio(self):
     def test_clock_gettime_monotonic(self):
         rv, out, _ = invoke_c_prog("clock_gettime_monotonic")
         assert rv == 0
-        dct = parse_keyvals(out)
+        dct = parse_keyvals(out, True)
         # Depends on speed of CPU, but should be very close to 1
         assert 0.95 <= dct["monotonic_delta"] <= 1.05
+
+    def test_msr_time(self):
+        rv, out, _ = invoke_c_prog("msr_time")
+        assert rv == 0
+        dct = parse_keyvals(out)
+
+        # I expect reading MSRs to be more than 70x slower than reading MSRs.
```

Comment:
The 70x is somewhat arbitrary, but I wanted to give it some room for manauver. We can try > and see what happens.

The test does not run on OpenBSD as we have no way to read core cycles yet, hence it is guarded:
```
@pytest.mark.skipif(not CORECYCLES_SUPPORT, reason="would not make sense")
```